packages/app/src/DashboardPage.tsx

```diff
@@ -200,14 +200,22 @@ const Tile = forwardRef(
           <div className="fs-7 text-muted">{chart.name}</div>
           <i className="bi bi-grip-horizontal text-muted" />
           <div className="fs-7 text-muted d-flex gap-2 align-items-center">
-            {hasAlert && (
-              <div
-                className="rounded px-1 text-muted bg-grey opacity-90 cursor-default"
-                title="Has alert"
-              >
-                <span className="bi bi-bell" />
+            {alert && (
+              <div className="rounded px-1 text-muted bg-grey opacity-90 cursor-default">
+                {alert.state === 'ALERT' ? (
+                  <i
+                    className="bi bi-bell text-danger effect-pulse"
```

Comment:
Super cool effect. It would be awesome to add this to sidebar bell as well (saved logs alert)